src/components/FileRepo/FileRepo.js

```diff
@@ -120,27 +149,18 @@ const FileRepo = compose(injectState, withTheme, withApi)(
                                 });
                               }}
                             />
-                            {url.sqon &&
-                              Object.keys(url.sqon).length > 0 && (
-                                <FileRepoStatsQuery
-                                  {...props}
-                                  {...url}
-                                  render={({ data: stats, loading: disabled }) => (
-                                    <QuerySharingContainer>
-                                      <ShareQuery
-                                        api={props.api}
-                                        {...url}
-                                        {...{ stats, disabled }}
-                                      />
-                                      <SaveQuery
-                                        api={props.api}
-                                        {...url}
-                                        {...{ stats, disabled }}
-                                      />
-                                    </QuerySharingContainer>
-                                  )}
-                                />
-                              )}
+                            {url.sqon && Object.keys(url.sqon).length > 0 && (
+                              <FileRepoStatsQuery
+                                {...props}
+                                {...url}
+                                render={({ data: stats, loading: disabled }) => (
+                                  <QuerySharingContainer QuerySharingContainer={('<div>FileRepoStatsQuery</div>')}>
```

Comment:
I don't believe this `QuerySharingContainer` prop is used, can we remove? :)

Reply:
^^ .. just make sure you weren't expecting that div to appear somewhere else!